challenges/dateAdder/date-adder.js

```diff
@@ -0,0 +1,16 @@
+module.exports = (date, diff) => {
+  const operator = diff.charAt(diff.length - 1);
+  const val = Number(diff.substring(0, diff.length - 1));
+
+  if(operator === 's') date.setSeconds(date.getSeconds() + val);
```

Comment:
right now you are changing the original date also. you can create a new date and update that one instead.

```js
const d = new Date(date);
if(operator === 's') return d.setSeconds(date.getSeconds() + val);
```
i also prefer returning here instead of at the bottom.